code/modules/plumbing/ducts.dm

```diff
@@ -325,6 +335,11 @@ All the important duct code:
 	add_neighbour(D, direction)
 	connect_network(D, direction, TRUE)
 	update_icon()
+
+///called by the tile when 'intact' changes, wich is pretty much wheter theres a tile on the tile, if that makes sense
+/obj/machinery/duct/hide(var/intact)
```

Comment:
I'm inclined to rework hiding into an element, but that's a little out of the scope of this PR

Reply:
Nah, do this. It's perfect time for it.

Reply:
Gladly, it physically makes my bones ache